awscli/customizations/wizard/commands.py

```diff
@@ -10,8 +10,73 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
-from awscli.customizations.wizard import devcommands
+from awscli.customizations.wizard import devcommands, factory
+from awscli.customizations.wizard.loader import WizardLoader
+from awscli.customizations.commands import BasicCommand
 
 
 def register_wizard_commands(event_handlers):
     devcommands.register_dev_commands(event_handlers)
+    loader = WizardLoader()
+    commands = loader.list_commands_with_wizards()
+    _register_wizards_for_commands(commands, event_handlers)
+
+
+def _register_wizards_for_commands(commands, event_handlers):
+    for command in commands:
+        event_handlers.register('building-command-table.%s' % command,
+                                _add_wizard_command)
+
+
+
+
+def _add_wizard_command(session, command_object, command_table, **kwargs):
+    runner = factory.create_default_wizard_runner(session)
+    cmd = WizardCommand(
+        session=session,
+        loader=WizardLoader(),
+        parent_command=command_object.name,
+        runner=runner,
+    )
+    command_table['wizard'] = cmd
+
+
+class WizardCommand(BasicCommand):
+    def __init__(self, session, loader, parent_command, runner,
```

Comment:
It might be good to fill out some of the `BasicCommand` class properties . Right now for the top level `wizard` command, the documentation I get is:
```
COMMANDNAME()                                                    COMMANDNAME()



NAME
       commandname -

DESCRIPTION
       describe the command

       See 'aws help' for descriptions of global parameters.

SYNOPSIS
          commandname

OPTIONS
       None

       See 'aws help' for descriptions of global parameters.
```
Same thing applies for the subcommand wizards.